packages/lesswrong/components/ea-forum/giving-portal/PreVoteButton.tsx

```diff
@@ -35,7 +39,7 @@ const PreVoteButton = ({vote, document, className, classes}: PreVoteProps & {
   const { captureEvent } = useTracking();
 
   const hasVoted = !!document.currentUserExtendedVote?.preVote;
-  const icon = hasVoted || hover ? "Heart" : "HeartOutline";
+  const icon = hasVoted || (hover && !isMobile()) ? "Heart" : "HeartOutline";
```

Comment:
There is a big disclaimer about using this function causing SSR mismatch, but this can't happen in this case because hover should always be false on page load.

(Also it's not clear whether using this or useScreenWidth is better, I went with this because I've previously found useScreenWidth to not work, and on narrow desktop screens we still want the hover 🤷 )

Reply:
Seems fine. We should really fix useHover though. There's no reason that useHover can't check for `!isMobile()`.